src/ocm/selectors/clusters.tsx

```diff
@@ -33,7 +34,7 @@ export const selectClustersUIState = createSelector(
 const clusterToClusterTableRow = (cluster: Cluster): IRow => {
   const { id, name = '', openshiftVersion, baseDnsDomain, createdAt } = cluster;
   const dateTimeCell = getDateTimeCell(createdAt);
-
+  const { t } = useTranslation();
```

Comment:
The IDE is highlighting an issue here (hook order), and when try to load the page, it fails. Maybe is because of the parameter order I mention below, but please check this.

Reply:
`clusterToClusterTableRow` is not a component